map.py

```diff
@@ -7,25 +7,23 @@
 
 from typing import Set, Dict, Tuple
 from dataclasses import dataclass, field
-from collections import defaultdict
 from copy import copy
 
 @dataclass
 class Place:
 	
+	tag: str
 	description: str
+	category: str
+	variant: str
```

Comment:
è davvero necessario che un place sappia che categoria e variante è?

Reply:
La categoria la uso per vedere se due luoghi possono essere collegati. La variante in effetti è un'informazione in più che non uso da nessuna parte.

Reply:
> La categoria la uso per vedere se due luoghi possono essere collegati

Immagino nella funzione per controllare se la mappa è valida?